riscv/src/compiler.rs

```diff
@@ -1332,6 +1333,18 @@ fn process_instruction(instr: &str, args: &[Argument]) -> Vec<String> {
                 rd,
             )
         }
+        "lhu" => {
+            let (rd, rs, off) = rro(args);
+            // TODO we need to consider misaligned loads / stores
+            only_if_no_write_to_zero_vec(
+                vec![
+                    format!("addr <== wrap({rs} + {off});"),
+                    format!("{rd} <== mload();"),
+                    format!("{rd} <== and(0x0000ffff, {rd});"),
```

Comment:
`lhu` = load half unsigned, and extend with 0.
the first two lines are the same as `load word`, we simply clear the 16 MSB after